Bottom: 977c55cf72e78bd66a5db0c9e01159785d5c76c1
Top:    42d98ebb7803eff19d3753409aec3c30727b499f
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-08-07 15:19:33 +0800

Refresh of net-core-introduce-xdp-rx

---

diff --git a/drivers/net/macvlan.c b/drivers/net/macvlan.c
index a22b2f4..740ac44 100644
--- a/drivers/net/macvlan.c
+++ b/drivers/net/macvlan.c
@@ -1494,11 +1494,6 @@ static int macvlan_changelink(struct net_device *dev,
 		if (vlan->mode == MACVLAN_MODE_SOURCE &&
 		    vlan->mode != mode)
 			macvlan_flush_sources(vlan->port, vlan);
-<<<<<<< current
-=======
-			vlan->port->source_count--;
-		}
->>>>>>> patched
 	}
 
 	if (data && data[IFLA_MACVLAN_FLAGS]) {
